src/styles/animation.scss

```diff
@@ -0,0 +1,9 @@
+.card:hover {
+  transform: scale(120%);
```

Comment:
The `transform: scale(120%);` property should use a decimal value for scaling, like `1.2`, instead of a percentage. This is a common practice for the `transform: scale()` function.